consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -155,12 +155,16 @@ public void run() {
             if (executors.isEmpty()) {
                 if (run && !Thread.interrupted()) {
                     MigrationJob nextJob = getNextAvailableJob();
-                    long timeUntilNextJob = DEFAULT_WAIT;
-                    if (nextJob != null) {
+                    long timeUntilNextJob = DEFAULT_JOB_WAIT_MSEC;
+                    if (nextJob != null && nextJob.getWhenAvailableDate() != null) {
                         timeUntilNextJob = nextJob.getWhenAvailableDate().getTime() - new Date().getTime();
                     }
+                    if (timeUntilNextJob > MAX_JOB_WAIT_MSEC) {
+                        timeUntilNextJob = MAX_JOB_WAIT_MSEC;
```

Comment:
I know that it might be documented already about this, but It would be nice to have a log entry here (warn level?) as someone can misuse migrator and get undesired behavior without anything being logged about it. What do you think?